configurator/src/main.rs

```diff
@@ -609,7 +644,7 @@ fn main() -> Result<(), anyhow::Error> {
         let output = std::process::Command::new("curl")
             .arg("--no-progress-meter")
             .arg("-X")
-            .arg("POST")
+            .arg("GET")
```

Comment:
What's this for?

Reply:
This solved the first time install issue we found when we first updated to v15. 
it was the reason you pulled it back off the beta marketplace. 

This creates an encoded aezeed cipher when making a new wallet for LND. the POST was never correct, but apparently LND didn't care until v15.

Reply:
Very interesting, thanks for the explanation!